pom.xml

```diff
@@ -19,7 +19,10 @@ the License.-->
   <groupId>io.cdap.tests.e2e</groupId>
   <artifactId>cdap-e2e-framework</artifactId>
   <version>0.0.1-SNAPSHOT</version>
-
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```

Comment:
why do we need to add these properties?

Reply:
a warning was shown while building the project, that encoding is not defined, and we were giving it manually on our VMs. So added the same